global.css

```diff
@@ -0,0 +1,16 @@
+* {
```

Comment:
Why do you have a submodule called "brent" in here?

Reply:
because my site is a portfolio and the submodule is his site.
so when i publish his site, i can just publish it without wrapping in my code

Reply:
why don't you make them two different sites?